src/Symfony/Component/Validator/Tests/Constraints/CountValidatorTest.php

```diff
@@ -209,12 +209,12 @@ public function testConstraintAnnotationDefaultOption()
     public function testDivisibleBy()
     {
         $constraint = new Count([
-            'divisibleBy' => 123,
+            'divisibleBy' => 3,
```

Comment:
The validated collection contains three elements. Checking whether it is divisible by 123 would result in a constraint validation error.

Reply:
It was intentional to highlight the fact that the `divisibleBy` option is not really tested here, ie that the value does not matter in this test.

Reply:
Let's close then? Or perhaps we can add a comment to explain?

Reply:
There is already a comment above the test. I will let you decide what is the best here.